conda-recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ requirements:
       - cython
       - cmake 3.19
       - dpctl >=0.10
-      - mkl-devel-dpcpp
+      - mkl-devel-dpcpp {{ MKL_VER }}
```

Comment:
You could provide default version of package with `environ.get`:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#jinja-templates